ResearchKit/ActiveTasks/ORKHealthQuantityTypeRecorder.m

```diff
@@ -52,20 +52,6 @@ @interface ORKHealthQuantityTypeRecorder () {
 
 @end
 
-#ifdef __IPHONE_10_0
```

Comment:
I submitted stripping this as a separate PR. https://github.com/ResearchKit/ResearchKit/pull/970